pkg/controller/catalog/initialize.go

```diff
@@ -483,6 +483,7 @@ func buildRuntimeBuilderImageSpectrum(options spectrum.Options) error {
 
 	_, err := spectrum.Build(options, directories...)
 	if err != nil {
+		Log.Error(err, "Error trying to build Camel K builder container")
```

Comment:
We are returning the error. The caller should know what to do with it, if printing a warning or what else.